### DISCUSSION TITLE
Java Script Q&A

### DISCUSSION THREAD

**Discussion by odinharindra** [odinharindra/training#5] [Q&A]
Discussions related to Java script can be asked here

**Comment by ODUnike** [comment]
Hello guys my first day here, would love to learn Java Script. Can someone point me in the right direction.

**Reply by AlyphNull** [reply]
Hi OD the best way to learn JS is through project heavy Curriculums. Try the fullstack course by IBM on Coursera.

**Comment by jaskaran916** [comment]
Hi @odinharindra where are the assignments located here and the portal, can't seem to find them
